unintended_ml_bias/new_madlibber/madlibber.py

```diff
@@ -0,0 +1,98 @@
+import re
+import csv
+
+from enums import Toxicity, WordType, WordConnotation, WordGender
+
+class Madlibber(object):
+  def __init__(self, path_helper):
+    self.path_helper = path_helper
+    self.__words = {}
+
+  def load_and_sanity_check_words(self):
+    f = open(self.path_helper.word_file, 'r')
+    csv_f = csv.reader(f)
+    next(csv_f)  # skip header
```

Comment:
fyi if you use the csv.DictReader class it will handle the header for you, creating a dict for each non-header row with keys equal to the header entries

Reply:
Interesting, thanks!